src/flow/domain/mod.rs

```diff
@@ -829,28 +868,41 @@ impl Domain {
                         }).unwrap();
                     }
                 }
+                mut m @ Packet::ReplayPiece {
+                    context: ReplayPieceContext::Partial { ignore: true, .. }, ..
+                } => {
+                    let mut n = self.nodes[&path.last().unwrap().0.as_local()].borrow_mut();
+                    if n.is_egress() && n.is_transactional() {
+                        // No need to set link src/dst since the egress node will not use them.
```

Comment:
"We need to propagate this replay even though it contains no data, so that downstream domains don't wait for its timestamp"